src/main/java/org/opensearch/knn/index/KNNVectorFieldMapperUtil.java

```diff
@@ -0,0 +1,141 @@
+/*
+ * SPDX-License-Identifier: Apache-2.0
+ *
+ * The OpenSearch Contributors require contributions made to
+ * this file be licensed under the Apache-2.0 license or a
+ * compatible open source license.
+ *
+ * Modifications Copyright OpenSearch Contributors. See
+ * GitHub history for details.
+ */
+
+package org.opensearch.knn.index;
```

Comment:
can we move this class to org.opensearch.knn.index.mapper package, as this class is only getting used in Mappers